docs/source/json-api/index.rst

```diff
@@ -32,127 +32,100 @@ We welcome feedback about the JSON API on `our issue tracker
+=============
 
-.. code-block:: shell
+The JSON API essentially performs two separate tasks:
+
+1. It talks to the IAM of the Ledger API to get data it needs to operate, for this it may need to *provide an access token to an IAM* that requires authentication.
+2. It handles requests coming from one or more Parties, for this each party needs to provide an *access token with each request* it sends to the JSON API.
+
+.. note:: By default sandboxes do not have an IAM and do not need the access token mentioned in point 1
+
+Ledger API Auth with IAM
+------------------------
+
+The IAM access token is used exclusively for maintaining the internal list of known packages and templates.
+
+.. note:: At no point should this access token be provided to an end user, these are for internal use only.
+
+Every IAM access token is different and will depend on what your specific ledger operator's IAM requires.
+The JSON API server requires no access to party-specific data, only access to the ledger identity and package services.
+A token issued for the HTTP JSON API server should contain enough claims to contact these two services but no more than that.
+Please refer to your ledger operator's documentation to find out how.
+
+Once you have retrieved your access token, you can provide it to the JSON API by storing it in a file
+and starting ``daml json-api`` with the flag ``--access-token-file /path/to/your/token.file``.
+
+If the token cannot be read from the provided path or the Ledger API reports an authentication error
+(for example due to token expiration), the JSON API will report the error via logging. 
+
+.. note:: If the token file is updated with a new token it will be picked up at the next attempt to send a request. You can use this to handle cases where an old token expires without restarting your JSON API service.
 
-    $ daml new iou-quickstart-java --template quickstart-java
-    $ cd iou-quickstart-java/
-    $ daml build
-    $ daml sandbox --wall-clock-time --ledgerid MyLedger ./.daml/dist/quickstart-0.0.1.dar
-    $ daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575
+Party Auth
+----------
 
-Choosing a party
-****************
+Every request from a client to the JSON API requires you to specify a party and some other settings,
+with a JWT token.  HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
 
-Every request requires you to specify a party and some other settings,
-with a JWT token.  Normal HTTP requests pass the token in an
-``Authentication`` header, while WebSocket requests pass the token in a
-subprotocol.
+.. note:: While the JSON API receives the token it doesn't handle it itself. Upon receiving a token it will pass it on to the Ledger API's AuthService which will then determine if the token is valid and authorized. As such each token format is ledger/implementation-specific and the below example reflects only the format that is used for the DAML Sandbox.
```

Comment:
```suggestion
.. note:: While the JSON API receives the token it doesn't validate it itself. Upon receiving a token it will pass it on to the Ledger API's AuthService which will then determine if the token is valid and authorized. However, the JSON API does decode the token to extract ledger id, application id and party so it requires that you use the JWT format documented here <insert link>.
```
It does decode the token for party-specific requests so it doesn’t actually work if your ledger requires a different token format.